kalite/settings.py

```diff
@@ -214,7 +214,7 @@ def package_selected(package_name):
     RESTRICTED_TEACHER_PERMISSIONS = True
     FIXED_BLOCK_EXERCISES = 5
     QUIZ_REPEATS = 3
-UNIT_POINTS = 2000
+    POINTS = 2000
```

Comment:
Shouldn't need this parameter at all, as we are not maxing out student points at any particular level.